packages/shortest/src/ai/snapshot-client.ts

```diff
@@ -0,0 +1,102 @@
+import { existsSync, readFileSync } from 'fs';
+import { join } from 'path';
+import { AIClient } from './client';
+import { BrowserTool } from '../browser/core/browser-tool';
+import { ActionInput } from '../types/browser';
+import { TestContext } from '../types/test';
+import Anthropic from '@anthropic-ai/sdk';
+import { sleep } from '@anthropic-ai/sdk/core';
+import pc from 'picocolors';
+
+const JUDGMENT_PROMPT = `You are a test result validator. Your task is to evaluate if a test passed or failed based on the final state.
```

Comment:
What is the difference between `JUDGMENT_PROMPT` and `JUGDEMENT_SYSTEM_PROMPT`. Preferably, move this to a new file in `prompts` dir and export from `index.ts`.

Reply:
Cursor just made it twice, fixing soon.